test/unit/units/setup_host_network_test.py

```diff
@@ -91,3 +98,80 @@ def test_main(
         fstab.export.assert_called_once_with(
             '/etc/system-root.fstab'
         )
+
+    @patch('suse_migration_services.units.setup_host_network.is_resolv_empty')
+    @patch('suse_migration_services.logger.Logger.setup')
+    @patch('suse_migration_services.command.Command.run')
+    @patch('suse_migration_services.units.setup_host_network.Fstab')
+    @patch('os.path.exists')
+    @patch('shutil.copy')
+    @patch('glob.glob')
+    @patch('os.path.isfile')
+    def test_main_no_copy(
```

Comment:
I think the intention of this test is to check that etc/resolv conf is not copied. I wonder about all the other assertions for fstab and so on

Reply:
I would have expected something like ```assert not mock_shutil_copy.called``` or ```assert call... not in mock_shutil_copy.call_args_list```. Something that is more explicit to what the test is good for. Does this makes sense ?

Reply:
All the other conditions of main are part of other test cases. So I think it's fine if this test just focuses on no copy happened in the most easy way to get that condition

Reply:
Fixed.